Makefile

```diff
@@ -59,14 +59,19 @@ build-e2e:
 
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
+	# Run Disruptive tests first. Those might include things causing all masters to restart, and should run serially
+
+	# TODO: Disabled, until all disruptive tests are prefixed with post-conditions, allowing cluster to recover
+	# hack/ci-integration.sh $(GINKGO_ARGS) -focus="Disruptive" || (hack/junitmerge.sh && exit 1)
+
 	# Run operator tests first to preserve logs for troubleshooting test
 	# failures and flakes.
 	# Feature:Operator tests remove deployments. Thus loosing all the logs
 	# previously acquired.
-	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Feature:Operators" || (hack/junitmerge.sh && exit 1)
-	hack/ci-integration.sh $(GINKGO_ARGS) -p -skip="Feature:Operators|Autoscaler" || (hack/junitmerge.sh && exit 1)
+	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Feature:Operators" -skip="Disruptive" || (hack/junitmerge.sh && exit 1)
```

Comment:
Should we be more explicit here and make this `[Disruptive]` including the brackets, just in case a test randomly has disruptive in the title anyway?

Reply:
I'm afraid of possibility for this to become a wildcard regular expression.

Reply:
Does the syntax make it a regex, In which case we can escape the brackets right?

Reply:
Let's try this. Need another LGTM then.